deepchecks/core/serialization/suite_result/widget.py

```diff
@@ -307,6 +312,340 @@ def prepare_unconditioned_results_summary(
             )
             return DataFrameSerializer(df.style.hide_index()).serialize()
 
+    def prepare_fixes(self) -> Widget:
+        """Prepare fixes section.
+
+        Returns
+        -------
+        ipywidgets.Widget
+        """
+        out = Output()
+
+        # TODO: Change the if from getattr to isinstance, we left it like this because we had circular imports
+        fixable_check_results = [check_result for check_result in self.value.get_not_passed_checks()
+                                 if getattr(check_result.check, 'fix', None) is not None]
```

Comment:
will look better to implement some Mixin in my opinion

Reply:
@nirhutnik we can change it to mixin, but we had some circular imports problems, i can spend some time to see if it works, but i think that if what we have now works we can leave it the way it is